packages/vite/src/node/server/middlewares/indexHtml.ts

```diff
@@ -360,6 +360,8 @@ function preTransformRequest(server: ViteDevServer, url: string, base: string) {
       return
     }
     // Unexpected error, log the issue but avoid an unhandled exception
-    server.config.logger.error(e.message)
+    server.config.logger.error(`Pre-transform error: ${e.message}`, {
+      timestamp: true,
```

Comment:
Maybe we could also take the opportunity to pass the error to `logger.error` so it is the same as the way from the other unexpected error above?
```suggestion
      error: e,
      timestamp: true,
```